ibm/service/power/resource_ibm_pi_image.go

```diff
@@ -360,17 +358,17 @@ func isIBMPIImageRefreshFunc(ctx context.Context, client *st.IBMPIImageClient, i
 		}
 
 		if image.State == "active" {
-			return image, helpers.PIImageActiveStatus, nil
+			return image, Arg_ImageActiveStatus, nil
 		}
 
-		return image, helpers.PIImageQueStatus, nil
+		return image, Arg_ImageQueStatus, nil
 	}
 }
 
-func waitForIBMPIJobCompleted(ctx context.Context, client *st.IBMPIJobClient, jobID string, timeout time.Duration) (interface{}, error) {
+func waitForIBMPIJobCompleted(ctx context.Context, client *instance.IBMPIJobClient, jobID string, timeout time.Duration) (interface{}, error) {
 	stateConf := &resource.StateChangeConf{
-		Pending: []string{helpers.JobStatusQueued, helpers.JobStatusReadyForProcessing, helpers.JobStatusInProgress, helpers.JobStatusRunning, helpers.JobStatusWaiting},
-		Target:  []string{helpers.JobStatusCompleted, helpers.JobStatusFailed},
+		Pending: []string{Arg_JobStatusQueued, Arg_JobStatusReadyForProcessing, Arg_JobStatusInProgress, Arg_JobStatusRunning, Arg_JobStatusWaiting},
```

Comment:
So these aren't arguments to the terraform resource, and the value of them should not be changed. Make a new section for the status constants. Something like:

```
Status_Queued = "queued"
Status_InProgress = "in_progress"
...
```